src/templates/blog-post.tsx

```diff
@@ -55,7 +58,14 @@ const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
       <SEO
         title={title}
         description={description}
-        image={picture}
+        image={
+          picture &&
+          (isProduction
+            ? `/blog/${picture.fields.sourcePath}`
+            : picture.gatsbyImageData)
+        }
+        imageHeight={picture?.gatsbyImageData.height}
+        imageWidth={picture?.gatsbyImageData.width}
```

Comment:
same sentiment here with passing down `picture` instead of disassembling it here, with a similar conclusion of it being a nitpick.

Reply:
Similiar to your previous comment, I set them like so to make the typescript less complex. But if you think it would be better, I'm happy to set the `HeroPic` and `SEO` to take the entire `picture` object!